FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -88,26 +95,25 @@
 	<owl:Class rdf:about="&fibo-fbc-dae-cre;DistressedRatingsDowngrade">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-cre;Downgrade"/>
 		<rdfs:label xml:lang="en">distressed ratings downgrade</rdfs:label>
-		<skos:definition xml:lang="en">credit event triggered when the credit rating of the reference obligation is downgraded to a distressed rating level or the rating is withdrawn by one or more ratings agency</skos:definition>
+		<skos:definition xml:lang="en">credit event triggered when the credit rating of a party or reference obligation is downgraded to a distressed rating level or the rating is withdrawn by one or more ratings agency</skos:definition>
```

Comment:
but the superclass Downgrade is ObligationSpecific so "of a party" does not apply.
I don't think we need "reference" before "obligation"
There seems to be a big difference between withdrawing a rating and providing a rating at level of "distressed". Should these not be separate event classes? See https://www.moodys.com/uploadpage/Mco%20Documents/SP13418%20Policy%20for%20Withdrawal%20of%20Credit%20Rating.pdf for reasons a rating could be withdrawn. It might not even be a Downgrade e.g. if due to a reorg